src/modules/add-site/components/blueprint-icon.tsx

```diff
@@ -7,53 +7,53 @@ export const BlueprintIcon = ( { size = 32 }: { size?: number } ) => (
 		xmlns="http://www.w3.org/2000/svg"
 	>
 		<path
-			d="M17.3334 19.8828C14.6667 19.8828 10.6667 21.2161 10.6667 26.5495"
+			d="M13.2434 14.7355C11.242 14.7355 8.23975 15.7362 8.23975 19.7392"
 			stroke="#3858E9"
-			strokeWidth="2"
-			strokeLinecap="round"
-			strokeLinejoin="round"
+			stroke-width="1.5011"
+			stroke-linecap="round"
+			stroke-linejoin="round"
```

Comment:
Let's use camelcase for these attributes.

```suggestion
			strokeWidth="1.5011"
			strokeLinecap="round"
			strokeLinejoin="round"
```

Using dashes produces errors in the console:
<img width="1726" height="476" alt="Screenshot 2025-08-14 at 06 25 25" src="https://github.com/user-attachments/assets/bf38dbb5-5d81-4454-9197-941d97f2ee98" />